flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java

```diff
@@ -77,14 +77,20 @@ public void testRedisListDataType() throws Exception {
     @Test
     public void testRedisSetDataType() throws Exception {
         DataStreamSource<Tuple2<String, String>> source = env.addSource(new TestSourceFunction());
-        RedisSink<Tuple2<String, String>> redisSink = new RedisSink<>(jedisPoolConfig,
+        RedisSink<Tuple2<String, String>> redisSaddSink = new RedisSink<>(jedisPoolConfig,
             new RedisCommandMapper(RedisCommand.SADD));
 
-        source.addSink(redisSink);
-        env.execute("Test Redis Set Data Type");
+        source.addSink(redisSaddSink);
+        env.execute("Test SADD");
 
         assertEquals(NUM_ELEMENTS, jedis.scard(REDIS_KEY));
 
+        RedisSink<Tuple2<String, String>> redisSremSink = new RedisSink<>(jedisPoolConfig,
```

Comment:
could we add a new test only to remove?

Reply:
> Could you separate in two commits one for each command added?

Sorry, It's a mistake, I'm new to github. 😅
I will close this PR and create a new one in future.

Thanks for review.